_visualisation/core/unknowns.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Unknowns
+layout: visualisation
+---
+
+Often, our picture of a beneficial ownership situation contains many unknowns. The following rules allow BOVS Diagrams to portray what we *do* know and indicate what we *don't*.
+{: .lead}
+
+
+## Unknown Parties
+
+**We may know that Parties exist, but *lack detail* about them.** For instance:
+
+* We may know that an intermediary exists in between two Entities, but have no information about it
+* We may know that an Entity must be owned by a Person, but not know whom
+* A Person may be anonymous
+  * For legitimate reasons
+  * Due to non-disclosure or missing data
+
+As a Party is known to exist – despite its details being unknown – we still draw a Node for it:
+
+* Use the Unknown variant of the icon for that type of Party (see below)
+* Label the node with an [Identifier](/visualisation/core/parties-nodes), using the status "Unknown" or "Anonymous"
+
+### Icons
+
+Each icon in the [Vocabulary](/visualisation/core/vocabulary) has an Unknown variant. In general, the Unknown variant of an icon is the outline of the original icon, with a question mark inside it.
+
+### Unknown Number
+
+**We may know that Parties exist, but not *how many*.** In such situations:
+
+* Show a dotted [Stack](/visualisation/core/stacks).
```

Comment:
On investigation, this is because the underline style is defined on `p a` so doesn't apply to `li`, however, we can't do `p, li { a ... }` as then this applies it to things like menu items. Really this should be scoped e.g. `.text-body a { ... }` and then that class applied to the `div`s that contain the main body of the page. I've done this for the visualisation section.